### PR TITLE
Task #223691 feat: Updated codes in state, districts and blocks and Delete dynamically 

### PR DESCRIPTION
**Pull request by rahulg1254**

### CODE REVIEW

File: src/pages/block.tsx [hunk 1]
```diff
@@ -473,7 +475,9 @@ const Block: React.FC = () => {
 
       <ConfirmationModal
         modalOpen={confirmationDialogOpen}
-        message={t("COMMON.ARE_YOU_SURE_DELETE")}
+        message={t("COMMON.ARE_YOU_SURE_DELETE", {
+          state: `${selectedStateForDelete?.block} Block`,
```
Comment (itsvick) [inline src/pages/block.tsx:479]: @rahulg1254 translate this `Block`

File: src/pages/district.tsx [hunk 2]
```diff
@@ -339,7 +339,9 @@ const District: React.FC = () => {
       />
       <ConfirmationModal
         modalOpen={confirmationDialogOpen}
-        message={t("COMMON.ARE_YOU_SURE_DELETE")}
+        message={t("COMMON.ARE_YOU_SURE_DELETE", {
+          state: `${selectedStateForDelete?.label} District`,
```
Comment (itsvick) [inline src/pages/district.tsx:343]: @rahulg1254 translate `District`

File: src/pages/block.tsx (hunk 1 above)
Comment (rahulg1254) [reply src/pages/block.tsx:479]: resolved

File: src/pages/district.tsx (hunk 2 above)
Comment (rahulg1254) [reply src/pages/district.tsx]: resolved

File: src/pages/block.tsx (hunk 1 above)
Comment (itsvick) [reply src/pages/block.tsx:479]: what did u resolved here? @rahulg1254

Comment (rahulg1254) [reply src/pages/block.tsx]: it is resolved now!!!

**Pull request merged**